ambry-frontend/src/test/java/com.github.ambry.frontend/AmbryBlobStorageServiceTest.java

```diff
@@ -2073,12 +2073,12 @@ private void injectAccountAndContainerForPutAndVerify(Container container, boole
 
     Container legacyContainerForPublicBlob =
         new ContainerBuilder(Container.DEFAULT_PUBLIC_CONTAINER_ID, "containerForLegacyPublicPut",
-            Container.ContainerStatus.ACTIVE, "This is a container for putting legacy public blob", false, false, false,
-            false, refAccount.getId()).build();
+            Container.ContainerStatus.ACTIVE, "This is a container for putting legacy public blob",
+            refAccount.getId()).setCacheable(true).build();
```

Comment:
looks like in initial code, `setCacheable` is false

Reply:
Yeah - was incorrect since `cacheable` should be `true` in public containers